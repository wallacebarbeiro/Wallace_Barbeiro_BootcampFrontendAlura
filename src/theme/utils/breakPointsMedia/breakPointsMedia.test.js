import breakpointsMedia from './index';

describe('breakpointsMedia()', () => {
  describe('when receives a argument', () => {
    test('and it has a only one breakpoint', () => {
      const breakpointsMediaResult = breakpointsMedia({ xs: { textAlign: 'center' } });
      expect(breakpointsMediaResult).toMatchSnapshot();
    });
  });

  // eslint-disable-next-line jest/no-identical-title
  describe('when receives a argument', () => {
    test('and it has two or more breakpoints', () => {
      const breakpointsMediaResult = breakpointsMedia({ xs: { textAlign: 'center' }, md: { textAlign: 'left' } });
      expect(breakpointsMediaResult).toMatchSnapshot();
    });
  });
});
