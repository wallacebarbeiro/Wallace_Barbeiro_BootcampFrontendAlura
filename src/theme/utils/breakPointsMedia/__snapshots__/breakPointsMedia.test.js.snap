// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`breakpointsMedia() when receives a argument and it has a only one breakpoint 1`] = `
Array [
  Array [
    "@media only screen and (min-width:",
    "0",
    "px){",
    "text-align: center;",
    "}",
  ],
]
`;

exports[`breakpointsMedia() when receives a argument and it has two or more breakpoints 1`] = `
Array [
  Array [
    "@media only screen and (min-width:",
    "0",
    "px){",
    "text-align: center;",
    "}",
  ],
  Array [
    "@media only screen and (min-width:",
    "768",
    "px){",
    "text-align: left;",
    "}",
  ],
]
`;
